### PR TITLE
Add post 50K users

### PR DESCRIPTION
**Pull request by beatrizserrano**
Need some inspiration here... ideas are welcome :)
I dated to the day after the party, we could include a few pics of that day?

### CODE REVIEW

**Comment by erxleben** [comment]
I would include the fact sheet here to show the recent numbers which are quite impressive. I have created a recent one: 
https://drive.google.com/file/d/1PFaUHtq0YVilCPWD34HrD4UZDClWsq9o/view?usp=sharing

**Comment by beatrizserrano** [comment]
> I would include the fact sheet here to show the recent numbers which are quite impressive. I have created a recent one: https://drive.google.com/file/d/1PFaUHtq0YVilCPWD34HrD4UZDClWsq9o/view?usp=sharing

that's a good idea, thanks! Could you put that in the branding repo? Remember also that we have the new factsheet here: https://github.com/usegalaxy-eu/branding/tree/master/factsheet

**Comment by beatrizserrano** [comment]
image

File: _posts/2022-06-23-reached-50000-users.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+Not only the amount of registered users have grown, the number of [tools installed](https://galaxyproject.eu/tools) is now over 2,800, 47M jobs have been run, 260K workflows have been executed... The training activities have also increased, reaching 13K trainees using [Training Infrastructure as a Service (TIaaS)](https://galaxyproject.eu/posts/2021/08/24/tiaas-flyer/) in [different aspects of science](https://galaxyproject.eu/news?tag=TIaaS). Take a look at these and other impressive numbers in our factsheet:
+
+<br>
+![factsheet](/assets/media/50K_users/factsheet_automatable_eu.png)
+<br><br>
+
+And we wanted to celebrate this important milestone with __all the people that have supported us__ in all sorts of ways: facility management, technical support from the compute center to manage an increasing [hardware infrastructure](https://galaxyproject.eu/news?tag=hardware), our sponsors, the EU office at the University of Freiburg, the Backofen lab, our power users, the [whole team](https://galaxyproject.eu/people) and the rest of the Galaxy community!
```
Comment (bgruening) [inline _posts/2022-06-23-reached-50000-users.md:44]: compute center could be linked and the uni-freiburg as well, backofen lab ...

**Comment by bgruening** [comment]
I think we should include the pulsar-partners as well, explicilty? and the site-partners, BE, FR ....

Thanks, looks great already!

File: _posts/2022-06-23-reached-50000-users.md (hunk 1 above)
Comment (erxleben) [reply _posts/2022-06-23-reached-50000-users.md:44]: add  Science Support center (Frank Krüger)?
Medical faculty?
MPI?

Mention support of Prorektor für Forschung ? @bgruening ?

**Pull request merged**